content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,403 @@
+for information on how to apply the custom defined `spire` template into `istio-proxy`. This allows for the CSI driver to mount the UDS on the sidecars.
+
+    Check Ingress-gateway pod state:
+
+    {{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          17s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
+    {{< /text >}}
+
+Data plane containers will only reach the `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
+Envoy will be able to fetch cryptographic identities from SPIRE.
+See [Register workloads](#register-workloads) to register entries for services in your mesh.
+
+## Register workloads
+
+This section describes the options available for registering workloads in SPIRE Server.
+
+### Option 1: Automatic registration using the SPIRE workload registrar
+
+By deploying [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar)
+along with SPIRE Server, new entries are automatically registered for each new pod that is created.
+
+See [Verifying that identities were created for workloads](#verifying-that-identities-were-created-for-workloads)
+to check issued identities.
+
+### Option 2: Manual Registration
+
+To improve workload attestation security robustness, SPIRE is able to verify against a group of selector values based on different parameters.
+
+1. To generate an entry for an Ingress Gateway with a set of selectors such as the
+   pod name and pod UID:
+
+    {{< text bash >}}
+$ INGRESS_POD=$(kubectl get pod -l istio=ingressgateway -n istio-system -o jsonpath="{.items[0].metadata.name}")
+$ INGRESS_POD_UID=$(kubectl get pods -n istio-system $INGRESS_POD -o jsonpath='{.metadata.uid}')
+    {{< /text >}}
+
+1. Get the spire-server pod:
+
+    {{< text bash >}}
+$ SPIRE_SERVER_POD=$(kubectl get pod -l app=spire-server -n spire -o jsonpath="{.items[0].metadata.name}")
+    {{< /text >}}
+
+1. Register an entry for the SPIRE Agent running on the node:
+
+    {{< text bash >}}
+$ kubectl exec -n spire $SPIRE_SERVER_POD -- \
+/opt/spire/bin/spire-server entry create \
+-spiffeID spiffe://example.org/ns/spire/sa/spire-agent \
+-selector k8s_psat:cluster:demo-cluster \
+-selector k8s_psat:agent_ns:spire \
+-selector k8s_psat:agent_sa:spire-agent \
+-node -socketPath /run/spire/sockets/server.sock
+
+Entry ID         : d38c88d0-7d7a-4957-933c-361a0a3b039c
+SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+Parent ID        : spiffe://example.org/spire/server
+Revision         : 0
+TTL              : default
+Selector         : k8s_psat:agent_ns:spire
+Selector         : k8s_psat:agent_sa:spire-agent
+Selector         : k8s_psat:cluster:demo-cluster
+    {{</ text >}}

```

Comment:
Lint error is caused by this:
```suggestion
    {{< /text >}}
```
You have a similar issue below.

Reply:
Nice catch, thanks.